Improvements to the Install DXP page (#2422)
ibexa/documentation-developer: docs/getting_started/install_ibexa_dxp.md

!!! note "Installing [[= product_name_oss =]]"

>     This installation guide details the steps to install [[= product_name =]] for users who have a subscription agreement with [[= product_name_base =]].

Reviewer: ```suggestion
    This installation guide shows in details how to install [[= product_name =]] for users who have a subscription agreement with [[= product_name_base =]].
```

!!! tip "Install Composer locally"

>     If you want to install Composer inside your project root directory only, follow the instructions on [installing Composer in the current directory](https://getcomposer.org/download/).

Reviewer: ```suggestion
    If you want to install Composer inside your project root directory only, follow the instructions for [installing Composer in the current directory](https://getcomposer.org/download/).
```
